apps/betterangels-backend/accounts/utils.py

```diff
@@ -45,3 +46,18 @@ def get_user_permission_group(user: Union[AbstractBaseUser, AnonymousUser]) -> P
         raise PermissionError("User lacks proper organization or permissions")
 
     return permission_group
+
+
+def get_outreach_authorized_users() -> Union[User, QuerySet[User]]:
```

Comment:
since we removed `interaction_author`, this function will only ever return a queryset
```suggestion
def get_outreach_authorized_users() -> QuerySet[User]:
```